x-pack/plugins/observability_solution/observability_onboarding/server/routes/flow/route.ts

```diff
@@ -245,8 +245,6 @@ const integrationsInstallRoute = createObservabilityOnboardingServerRoute({
       savedObjectId: params.path.onboardingId,
       observabilityOnboardingState: {
         ...savedObservabilityOnboardingState,
-        type: 'logFiles',
-        progress: {},
```

Comment:
Removing for now as integrations install happens later in the flow and this overrides the existing `progress`. There is [a separate task](https://github.com/elastic/kibana/issues/183363) to add additional progress tracking for install integrations.